doc/manual/customizing.rst

```diff
@@ -236,9 +236,26 @@ workspace for creating channels, and the "dialogs" workspace is where
 most local plugins are instantiated (when activated), by default.
 These two names should at least be defined somewhere in default_layout.
 
-==========================
-Adding or Removing Plugins
```

Comment:
@ejeschke , I find this title misleading, as the section does not mention anything on how to remove a plugin (is removing a plugin even possible?), so I modified it.

Reply:
@pllim, I think the only way to "remove" a plugin is to disable it with `--disable-plugins` command line option or setting of the `disable_plugins` setting in `general.cfg`.  Probably something which should be rectified. 

We could add a little text about these options.  I think global plugins will be auto started if they appear in the `--modules` command line option, but not local plugins.

Reply:
Otherwise this looks like a helpful addition to the docs.

Reply:
@ejeschke , setting `disable_plugins` in `general.cfg` does not work but using `--disable-plugins` in command line does. FYI.

Reply:
> setting disable_plugins in general.cfg does not work but using --disable-plugins in command line does

@pllim, did you confirm that using `disable_plugins` in `general.cfg` is working in dev?

Reply:
@ejeschke , it is confirmed working. Thanks!